config/locales/views/graders/en.yml

```diff
@@ -3,6 +3,7 @@ en:
   graders:
     actions:
       assign_grader: Assign grader(s)
+      randomly_assign: Randomly Assign Graders
```

Comment:
Ah, you don't need to add this, it's fine to have the button show the text of `randomly_assign_graders` below.